app/services/transaction_service/transaction.rb

```diff
@@ -48,22 +48,14 @@ def can_start_transaction_braintree(community_id:, author_id:)
   end
 
   def can_start_transaction_paypal(community_id:, author_id:)
-    person_account_response = paypal_account_api.get(
-      community_id: community_id,
-      person_id: author_id
-    )
-    personal_account_active = Maybe(person_account_response)[:data][:active]
-
-    admin_account_response = paypal_account_api.get(
-      community_id: community_id
-    )
-    admin_account_active = Maybe(admin_account_response)[:data][:active]
+    personal_account_prepared = PaypalHelper.account_prepared_for_user?(author_id, community_id)
```

Comment:
Hmmm... I'd like to get rid of some of the use of PaypalHelper around the code. Could the account object returned from the API have some enum field that indicates if the account is ready for action? The logic of what it means for an account to be prepared is logically owned by paypal service that manages the accounts.